src/IO/AssimpLoader/AssimpFileLoader.cpp

```diff
@@ -11,6 +11,15 @@
 #include <IO/AssimpLoader/AssimpLightDataLoader.hpp>
 
 #include <iostream>
+#if __has_include(<filesystem>)
+include_filesystem
+#  include <filesystem>
+#  define my_filesystem std::filesystem
```

Comment:
Do not #define aliases. Use ``using RaFilesystem=std::filesystem`` instead (or a name more simple if the aim of this alias is to simplify writing, e.g ``using stdfs=std::filesystem``) .
As this must be repeated in each file that need to acces ``std::filesystem``functionalities, put this include setting in one single file, e.g. ``Core/File/stdfilesystem.hpp``